Update Dependabot options reference for clarity (#40522)
github/docs: content/code-security/dependabot/working-with-dependabot/dependabot-options-reference.md

|--------|--------|
| `auto` | Default behavior.|
| `increase`| Always increase the minimum version requirement to match the new version. If a range already exists, typically this only increases the lower bound. |

> | `increase-if-necessary` | If the existing rule already allows the new version, Dependabot leaves the manifest file unchanged, but will still update the lockfile to use the latest version. If the existing rule does not allow the new version (for example, when a new major release falls outside the range), Dependabot will also update the manifest constraint. |

Reviewer: ```suggestion
| `increase-if-necessary` | Leave the version requirement unchanged if it already allows the new release (Dependabot still updates the resolved version). Otherwise widen the requirement. |
```
